src/LandPlanner.cpp

```diff
@@ -118,9 +121,26 @@ bool LandPlanner::getTargetTwist(const ros::Time& time,
             return false;
         }
         else if(anyPressed(landing_gear_message_)) {
-            state_ = LandState::DONE;
```

Comment:
why do we need a separate state for disarm? can't we just go straight into disarming the drone here once we know we are on the ground?

Reply:
It just felt weird to have a post-impact action in a state called "brace for impact". I can take it out but I liked the symmetry it had with the takeoff controller and also I don't think run time is an issue once we've landed